src/components/Schedule/ReservationInfo.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from "react";
+
+interface IProps {
+  date: string;
+  therapyName: string;
+  therapeutName: string;
+  place: string;
+  therapyLength: number;
+  price: number;
+}
+
+export const ReservationInfo: FC<IProps> = ({
+  date,
+  therapyName,
+  therapeutName,
+  place,
+  therapyLength,
+  price
+}) => (
+  <div className="pt-16">
+    <div className="flex font-gilbert">
+      <div className="text-21 border-solid pt-px pr-8 pb-8 border-lilac border-r">
+        {date}
+      </div>
+      <div className="text-21 ml-8 mt-px">{therapyName}</div>
+    </div>
+    <div className="flex flex-row items-center font-lato">
+      <div
+        style={{ marginLeft: "54px" }}
```

Comment:
create tailwind class for this

Reply:
the classes that Oleh needs, are already included in my pr